src/test/java/SearchServicePaginationNextPageTest.java

```diff
@@ -1,3 +1,24 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
please remove these three pagination classes and add one corresponding this https://github.com/t-karasova/grs-samples-python/blob/master/search/search_with_pagination.py one instead